cli/src/semgrep/git.py

```diff
@@ -172,6 +172,7 @@ def _get_git_status(self) -> GitStatus:
             "--ignore-submodules",
             "--relative",
             self._base_commit,
+            "--",  # end of options, start of file paths
```

Comment:
I've confirmed we can append other flags after this (eg, the `--merge-base` flag below. This param just needs to follow the branch name param (`self._base_commit`).